src/simulation.hpp

```diff
@@ -703,6 +708,14 @@ template <typename problem_t> void AMRSimulation<problem_t>::evolve()
 			WritePlotFile();
 		}
 
+		// Writing Plot files at time intervals
+		double next_plot_file_time = next_plot_file_num * plotTimeInterval_;
+		if (plotTimeInterval_ > 0 && ((next_plot_file_time > cur_time) && (next_plot_file_time < (cur_time + dt_[0])))) {
+			next_plot_file_num += 1;
+			WritePlotFile();
+			WriteCheckpointFile();
```

Comment:
The checkpoint files can be very large. I think we should not automatically output the checkpoint file at the same time. Perhaps make that a separate option?

Reply:
Sure thing.